tests/test.js

```diff
@@ -23,23 +23,53 @@ describe("People", () => {
     });
     it("should create a people entry with valid input", (done) => {
       // your code goes here
-      done();
+      chai
+        .request(app)
+        .post("/api/v1/people")
+        .send({ name: "Susan", age: 10 })
+        .end((err, res) => {
+          res.should.have.status(201);
+          res.body.should.contain({ message: "The person added" });
+          this.lastIndex = res.body.index;
+          done();
+        });
     });
   });
   describe("get /api/v1/people", () => {
     it(`should return an array of person entries of length ${this.lastIndex + 1}`, (done) => {
            // your code goes here
-           done();
+      chai
+        .request(app)
+        .post("/api/v1/people")
```

Comment:
I think this one is failing because you're using the wrong endpoint here. I think you want to call the endpoint that returns the person array.

Reply:
Good day. Please check the push request with corrected codes